src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -42,81 +67,66 @@ const CONTEXT_ACTIONS = [
         },
     },
 
-    // Copy chat link
     {
         text: 'Copy Link',
         icon: LinkCopy,
-        shouldShow: false,
+        shouldShow: () => false,
         onPress: () => {},
     },
 
-    // Mark as Unread
     {
         text: 'Mark as Unread',
         icon: Mail,
-        shouldShow: false,
+        shouldShow: () => false,
         onPress: () => {},
     },
 
-    // Edit Comment
     {
         text: 'Edit Comment',
         icon: Pencil,
-        shouldShow: false,
+        shouldShow: () => false,
         onPress: () => {},
     },
 
-    // Delete Comment
     {
         text: 'Delete Comment',
         icon: Trashcan,
-        shouldShow: false,
-        onPress: () => {},
+        shouldShow: props => props.reportAction.actorEmail === props.session.email,
+        onPress: (reportAction, reportID) => { deleteReportComment(reportID, reportAction); },
     },
 ];
 
-const propTypes = {
-    // The ID of the report this report action is attached to.
-    // eslint-disable-next-line react/no-unused-prop-types
-    reportID: PropTypes.number.isRequired,
+class ReportActionContextMenu extends React.Component {
```

Comment:
IMO this doesn't really need to be a class component, can be functional instead.